docs/book/src/networking-introduction.md

```diff
@@ -0,0 +1,98 @@
+|  [Control Flow, Congestion Avoidance/Control][5] | Yes  | No                       |  Yes, and user controlled                                          
+|  Based On | [IP][3]                | [IP][3]            |  UDP
+
+Unreliability gives great uncertainty with a lot of freedom, while reliability gives great certainty with costs in speed and freedom.
+That is why some protocols such as QUIC, SCTP are built on UDP instead of TCP. 
+UDP gives the end-user more control over the transmission than TCP can do. 
+While QUIC is build on top of UDP it does provides the same and even more features than TCP.
+
+## 2. Issues with TCP 
+
+While TCP has been around for long it does have some issues that QUIC tries to solve.
+
+### Head-of-line Blocking
+
+One of the biggest issues with TCP is that of Head-of-line blocking. 
+It is a convenient feature because it ensures that all packages are sent and arrive in order. 
+However, in cases of high throughput (multiplayer game networking) and big load in a short time (web page load), this can be catastrophic to your application performance.
+
+The issue is demonstrated in the following animation:
+
+![Head of line blocking][animation] 
+
+This animation shows that if a certain packet drops in transmission, all packets have to wait at the transport layer until it is resent by the other end.
+If the dropped packet is resent and arrived then all packets are freed from the transport layer. 
+
+Let's look at two areas where this head-of-line blocking issue is a huge deal.
+
+**Web Networking**
+
+The World Wide Web is a place where quick web-page load speeds are important (who wants to wait 200ms to the long right?).
+As websites get bigger and attention decreases, we need faster loading times for websites.
+
+To tackle this issue, HTTP-2 introduced a technique called multiplexing. 
+In short, this means that multiple TCP streams are initialized to communicate with the server. 
+Then If one of them blocks the whole website can continue to load seemingly while that single stream is retransmitting.
+
+We will take a deeper dive into this subject when looking at QUIC multiplexing.
+
+**Multiplayer Game Networking**
+
+The web space is not the only area where this head-of-line blocking is a big issue.
+Multiplayer action games are based on a constant stream of packets sent at a speed ranging from 10 to 30 packets per second.
+For the most part, the data in these packages are so time-sensitive that only the most recent data is useful.
+You can think of the input and position of the player, the orientation and speed, and the state of the physical objects in the world.
+If a single packet drops out we can not afford to queue up 10-30 packets a second until the lost packet is retransmitted. 
+This could cause annoying lag behavior and a bad user experience. 
+   
+### Connection Setup Duration
+
+In the standard HTTP+TLS+TCP stack, TCP needs a handshake to establish a session between server and client, 
```

Comment:
I don't think it makes sense to compare to TCP + TLS 1.2 here, so I only think this section is useful if there are improvements relative to a TCP + TLS 1.3 scenario -- otherwise, there's no strong reason to use QUIC.

Reply:
I rewrote it and included 1.3 in the reasoning.